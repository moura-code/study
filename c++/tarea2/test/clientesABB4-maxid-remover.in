crearClientesABB
crearCliente 7 Zenobia Gomez 16
agregarAClientesABB
maxIdClientesABB
crearCliente 21 Casimira Lopez 42
agregarAClientesABB
maxIdClientesABB
crearCliente 15 Valquiria Gomez 35
agregarAClientesABB
maxIdClientesABB
crearCliente 40 Zafiro Martinez 20
agregarAClientesABB
maxIdClientesABB
removerDeClientesABB 21
maxIdClientesABB
removerDeClientesABB 40
maxIdClientesABB
crearCliente 40 Zafiro Martinez 20
agregarAClientesABB
crearCliente 21 Casimira Lopez 42
agregarAClientesABB
crearCliente 25 Valquiria Gonzalez 19
agregarAClientesABB
maxIdClientesABB
crearCliente 45 Valquiria Garcia 33
agregarAClientesABB
imprimirClientesABB
maxIdClientesABB
removerDeClientesABB 40
imprimirClientesABB
removerDeClientesABB 25
imprimirClientesABB
liberarClientesABB
Fin
