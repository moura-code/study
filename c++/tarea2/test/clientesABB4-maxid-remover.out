Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> El clientesABB ha sido creado de forma exitosa.
2> Cliente creado de forma exitosa.
3> Se ha agregado el cliente al clientesABB de forma exitosa.
4> El mayor id en el clientesABB es 7.
5> Cliente creado de forma exitosa.
6> Se ha agregado el cliente al clientesABB de forma exitosa.
7> El mayor id en el clientesABB es 21.
8> Cliente creado de forma exitosa.
9> Se ha agregado el cliente al clientesABB de forma exitosa.
10> El mayor id en el clientesABB es 21.
11> Cliente creado de forma exitosa.
12> Se ha agregado el cliente al clientesABB de forma exitosa.
13> El mayor id en el clientesABB es 40.
14> El cliente con id 21 se removio del clientesABB.
15> El mayor id en el clientesABB es 40.
16> El cliente con id 40 se removio del clientesABB.
17> El mayor id en el clientesABB es 15.
18> Cliente creado de forma exitosa.
19> Se ha agregado el cliente al clientesABB de forma exitosa.
20> Cliente creado de forma exitosa.
21> Se ha agregado el cliente al clientesABB de forma exitosa.
22> Cliente creado de forma exitosa.
23> Se ha agregado el cliente al clientesABB de forma exitosa.
24> El mayor id en el clientesABB es 40.
25> Cliente creado de forma exitosa.
26> Se ha agregado el cliente al clientesABB de forma exitosa.
27> Cliente Zenobia Gomez
Id: 7
Edad: 16
Cliente Valquiria Gomez
Id: 15
Edad: 35
Cliente Casimira Lopez
Id: 21
Edad: 42
Cliente Valquiria Gonzalez
Id: 25
Edad: 19
Cliente Zafiro Martinez
Id: 40
Edad: 20
Cliente Valquiria Garcia
Id: 45
Edad: 33
28> El mayor id en el clientesABB es 45.
29> El cliente con id 40 se removio del clientesABB.
30> Cliente Zenobia Gomez
Id: 7
Edad: 16
Cliente Valquiria Gomez
Id: 15
Edad: 35
Cliente Casimira Lopez
Id: 21
Edad: 42
Cliente Valquiria Gonzalez
Id: 25
Edad: 19
Cliente Valquiria Garcia
Id: 45
Edad: 33
31> El cliente con id 25 se removio del clientesABB.
32> Cliente Zenobia Gomez
Id: 7
Edad: 16
Cliente Valquiria Gomez
Id: 15
Edad: 35
Cliente Casimira Lopez
Id: 21
Edad: 42
Cliente Valquiria Garcia
Id: 45
Edad: 33
33> ClientesABB liberado con exito.
34> Fin.
